lib/std/Target.zig

```diff
@@ -1007,15 +1019,19 @@ pub const Cpu = struct {
         riscv32,
         riscv64,
         sparc,
-        sparc64,
         sparcel,
+        sparc64,
+        sparc64el,
```

Comment:
This is actually very bizarre. SPARC v9 (64-bit) supports both little and big endian; SPARC v8 does not. I have no idea why we have a `sparcel` (32-bit?) tag. Needs cleanup.